### PR TITLE
app-crypt/nitrokey-app: version bump

### PR DESCRIPTION
**Pull request by nemunaire**
There are a lot of changes since the last bump in the tree.

Upstream now uses submodules, I don't know if the way I get them is very portage compliant.

### CODE REVIEW

File: app-crypt/nitrokey-app/files/0001-cmake-CXX-not-found.patch [hunk 1]
```diff
@@ -0,0 +1,11 @@
+--- a/libnitrokey/CMakeLists.txt	2017-08-06 01:35:11.382608018 +0200
++++ b/libnitrokey/CMakeLists.txt	2017-08-06 01:35:17.789623109 +0200
+@@ -3,8 +3,6 @@
+     OPTION(USE_CLANG "Use CLang" FALSE)
+     IF(USE_CLANG)
+         set(CMAKE_CXX_COMPILER "/usr/bin/clang++" CACHE string "clang++ compiler" FORCE)
+-    ELSE()
+-        set(CMAKE_CXX_COMPILER)
+     ENDIF()
```
Comment (SoapGentoo) [inline app-crypt/nitrokey-app/files/0001-cmake-CXX-not-found.patch]: remove this whole option please, it is awful (If I want to use Clang with the nitrokey-app, I will specify it via `CXX`)

**Comment by nemunaire** [comment]
I updated the patch, to remove the clang option. The system should be able to select the compiler via the CXX flag.

**Comment by nemunaire** [comment]
As it installs new icons in `/usr/share/icons`, I added calls to `gnome2_icon_cache_update`.

**Comment by monsieurp** [comment]
@SoapGentoo ping. I think this is ready for prime time.

**Comment by SoapGentoo** [comment]
1.3 is in tree now, which has only been possible due to a massive unbundling effort.

**Pull request closed without merging**